CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+#Contributing
+
+Persons contributing to this project agree to the [code of conduct](./CONDUCT.md) and agree to the potential redistribution of their code under our license.
+
+##Contributor Agreement: 
+
+###Task:
+- Ha @hadinh1306 : AIC + Table output
+- Simran @simrnsethi : BIC
+- Stephanie @rq1995 : Mallow’s 
+
+###Communication:
+- Responsive on Slack , especially during Friday and Saturday. 
+- 10-minute stand-up meeting on Monday during lunch break, to talk about weekly plan.
+
+### Other pull requests
+- We love pull requests from everyone.
+- Push to your fork and submit a pull request.
```

Comment:
For small changes, create `Branch` is ok. Guidelines below.
For big changes, please use `Folk`. Guidelines below.
Commit frequently with clear and concise commit message.
Write tests for function changes.
Once your changes are done, submit a Pull Request. 
Wait for comments from team member. Then make changes from your Branch/Folk repo. 
Only merge after reaching whole team agreement.